commitizen/defaults.py

```diff
@@ -104,7 +104,7 @@ class Settings(TypedDict, total=False):
 MINOR = "MINOR"
 PATCH = "PATCH"
 
-bump_pattern = r"^(((BREAKING[\-\ ]CHANGE|feat|fix|refactor|perf)(\(.+\))?(!)?)|\w+!):"
+bump_pattern = r"^((BREAKING[\-\ ]CHANGE|\w+)(\(.+\))?!?):"
```

Comment:
Why do we remove `feat|fix|refactor|perf)`

Reply:
As I understood commitizen allows you to define custom commit types, if we'd limit this to only `feat|fix|refactor|perf` then it wouldn't match for custom types.

Previously `\w+!` was included, but this didn't include the scope, the new regex is basically a simplified version that allows `\w+!` with a scope.

Reply:
Why? This change should be backwards compatible and it does fix a bug. `chore!` updates the major version but `chore(scope)!` does not.

Reply:
yep, I think you're right. looks good to me. thanks for helping out!